src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,164 @@
+package duke.main;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import duke.command.AddCommand;
+import duke.command.ChangeMarkCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidDateFormatException;
+import duke.exception.InvalidParametersException;
+import duke.exception.MissingCommandException;
+import duke.exception.MissingParametersException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+
+/** Deals with making sense of user command, manages error handling based on user input */
+public class Parser {
+    private static String[] splitText;
```

Comment:
Is there another way to name `splitText` as it may sound to the reviewer as a verb. Perhaps `formattedText`?

Reply:
Hmmm, sounds good, but I would like to make this variable obvious that it contains the split form of the string. I initially thought of ``splittedText`` instead of ``splitText`` but then realized that the past tense of split is split 🤔.